### PR TITLE
upgrade write-trimmomatic

### PR DESCRIPTION
**Pull request by mr-c**
Please review @ctb @SherineAwad

### CODE REVIEW

**Comment by SherineAwad** [comment]
--I tested it, and the lines where quotes are removed fail. Does it work with you? 

--Also, my main problem with this function when I used it is that if the user has in the directory files named like this:
R11.fastq.gz R12.fastq.gz  R21.fastq.gz  R22.fastq.gz , or something similar, it won't work or it will miss files up. So either update the code or we can say somewhere explicitly that this function won't accept such names.

**Comment by mr-c** [comment]
@SherineAwad good catch, I don't know what happened to those quote marks.

Do you have a fix for the R1 renaming problem?

**Comment by SherineAwad** [comment]
I did not fix this. I used to consider that this function won't accept such names. But I can work on that if you like.

File: sandbox/write-trimmomatic.py [hunk 1]
```diff
@@ -16,12 +17,12 @@
...
+interleave-reads.py s1_pe s2_pe | gzip -9c > ../%s
 
-cat s1_se s2_se | gzip -9c > ../%s
+cat s1_se s2_se | gzip -9c > ../../trimmed/%s
 cd ..
 rm -r ./trim/
 
 chmod u-w %s %s
-""" % (r1, r2, final_pe, final_se, final_pe, final_se)
+""" % (r1, r2, sys.argv[1], final_pe, final_se, final_pe, final_se)
```
Comment (SherineAwad) [inline sandbox/write-trimmomatic.py]: @mr-c  Why we need this "sys.argv[1]"? I think also function doesn't take arguments?

File: sandbox/write-trimmomatic.py [hunk 2]
```diff
@@ -16,12 +17,12 @@
     print """\
 mkdir trim
 cd trim
-java -jar /usr/local/bin/trimmomatic-0.30.jar PE ../%s ../%s s1_pe s1_se s2_pe s2_se ILLUMINACLIP:/usr/local/share/adapters/TruSeq3-PE.fa:2:30:10
-/usr/local/share/khmer/scripts/interleave-reads.py s1_pe s2_pe | gzip -9c > ../%s
+TrimmomaticPE ../%s ../%s s1_pe s1_se s2_pe s2_se ILLUMINACLIP:%s:2:30:10
+interleave-reads.py s1_pe s2_pe | gzip -9c > ../%s
 
-cat s1_se s2_se | gzip -9c > ../%s
+cat s1_se s2_se | gzip -9c > ../../trimmed/%s
```
Comment (SherineAwad) [inline sandbox/write-trimmomatic.py]: @mr-c  What is trimmed?

File: sandbox/write-trimmomatic.py [hunk 3]
```diff
@@ -16,12 +17,12 @@
     print """\
 mkdir trim
 cd trim
-java -jar /usr/local/bin/trimmomatic-0.30.jar PE ../%s ../%s s1_pe s1_se s2_pe s2_se ILLUMINACLIP:/usr/local/share/adapters/TruSeq3-PE.fa:2:30:10
-/usr/local/share/khmer/scripts/interleave-reads.py s1_pe s2_pe | gzip -9c > ../%s
+TrimmomaticPE ../%s ../%s s1_pe s1_se s2_pe s2_se ILLUMINACLIP:%s:2:30:10
```
Comment (SherineAwad) [inline sandbox/write-trimmomatic.py]: I think I got what you wanna do, but that's not working I think.

Comment (SherineAwad) [inline sandbox/write-trimmomatic.py]: TrimmpmaticPE? Not working I think.

**Comment by SherineAwad** [comment]
=Function now takes arguments, when done, I will update that in the metagenome protocols. Others will need to too. 
=Usually the protocol has a working directory and this ../../trimmed will throw the output outside and user might be blind of this. 
=Other than that LGTM :)

**Pull request closed without merging**